modular_skyrat/modules/modular_vending/code/clothesmate.dm

```diff
@@ -44,7 +44,7 @@
 				/obj/item/clothing/glasses/thin = 5,
 				/obj/item/clothing/glasses/hud/ar/projector = 5,
 				/obj/item/clothing/neck/face_scarf = 5,
-				/obj/item/clothing/gloves/bracer/wraps,
+				/obj/item/clothing/gloves/wraps, //BUBBER EDIT (removed "bracer" path before "wraps")
```

Comment:
If you just modularly override the armor values you don't have to do any of this